Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -20,6 +20,35 @@ public override SceneQueryType SceneQueryType
             set => raycastMode = value;
         }
 
+        [SerializeField]
+        [Min(0.0f)]
+        [Tooltip("Amount to pull back the center of the sphere behind the hand")]
```

Comment:
Make sure to communicate that this is only for purposes of showing / hiding hand ray, not for actually doing the grab detection.